packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -308,11 +324,41 @@ export default class TabAdvanced extends Component {
     getVmsHaValues().then(vmsHaValues => this.setState({ vmsHaValues }))
   }
 
+  _getCpuMask = createSelector(
+    () => this.props.vm.cpuMask,
+    cpuMask => {
+      if (cpuMask === undefined) {
+        return
+      }
+      return cpuMask.map(number => ({
+        value: `${number}`,
+        label: `Core ${number} `,
+      }))
```

Comment:
Why do you need objects here?

Reply:
Now we have changed the behavior of  `editable/Select`